src/main/java/reposense/parser/GroupConfigCsvParser.java

```diff
@@ -1,6 +1,6 @@
 package reposense.parser;
```

Comment:
Line 75 of this file, change `Otherwise adds...` to `Otherwise, adds...`.

Also move the `results` variable in line 80 to line 79.